packages/models/mock/Person/anthony-stark.json

```
@@ -0,0 +1,38 @@
+{
+  "id": "anthony-stark",
+  "type": "Person",
+  "attributes": {
+    "givenName": "Anthony",
+    "familyName": "Stark",
+    "additionalName": "Iron Man",
+    "gender": "male"
+  },
+  "relations": {
+    "alumniOf": [
+      {
+        "id": "mit",
+        "type": "Organizations"
+      }
+    ],
+    "affilation": [
```

Rename to `isMemberOf` for both `affiliation` and `alumni`

```suggestion
    "isMemberOf": [
```